jenkins-integration/jenkins-jobs/common/scripts/jenkins/pipeline.groovy

```diff
@@ -12,31 +61,42 @@ def step000_provision_sut(SKIP_PROVISIONING, script_dir) {
     }
 }
 
-def step010_setup_beaker(script_dir) {
-    withEnv(["SUT_HOST=${SUT_HOST}"]) {
+def step010_setup_beaker(script_dir, server_version) {
+    if (server_version["type"] != "pe") {
+        error "Unsupported server type: ${server_version["type"]}"
+    }
+
+    withEnv(["SUT_HOST=${SUT_HOST}",
+             "pe_version=${server_version["pe_version"]}",
```

Comment:
In pe-p-s-e jobs, I've seen values set like:

* pe_family = 2016.3
* pe_version = 2016.3.0-rc3-11-gbbfedf2

Will that work with how it looks like these jobs currently appear to set those to the same value?